providers/hubspot/read.go

```diff
@@ -60,8 +59,14 @@ func (c *Connector) Read(ctx context.Context, config common.ReadParams) (*common
 	} else {
 		// If NextPage is not set, then we're reading the first page of results.
 		// We need to construct the query and then make the request.
-		relativeURL := strings.Join([]string{"objects", config.ObjectName, "?" + makeQueryValues(config)}, "/")
```

Comment:
If there's a line of code to blame for yesterday's outage, this is the line. Notice how it's encoding the query param directly in to the string? And then that string gets passed to getURL? Well getURL already has its own handling for query params, and so the encoded path+query param being re-encoded likely caused issues. Either there were 2 question marks, which isn't technically valid, or it escaped the question mark - which would have made the object name itself invalid. Which actually seems more like what happened.